src/main/webapp/static/academic/css/dataTables.responsive.css

```diff
@@ -0,0 +1,94 @@
+table.dataTable.dtr-inline.collapsed tbody td:first-child,
```

Comment:
This extension was included based on this documentation:
https://datatables.net/extensions/responsive/examples/styling/bootstrap.html